buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -20,6 +20,7 @@ PUBLISH_TO_GITHUB_DEFAULT = PUBLISH_PACKAGES_DEFAULT
 // If required, switch off the mantiddocs conda package build. This is deliberately not listed in the
 // pipeline parameters because it's only switched off for ornl-next anaconda publishing.
 BUILD_MANTIDDOCS_PACKAGE = true
+WORKBENCH_INCLUDE_DOCS = true
```

Comment:
I think we can just use the `BUILD_MANTIDDOCS_PACKAGE` bool instead of creating this new one. It was added specifically for this use case anyway. The new bool will always have the same value so is redundant.